pcdsdevices/epics_motor.py

```diff
@@ -177,17 +177,27 @@ def format_status_info(self, status_info):
 """
 
     @property
-    def limits(self):
+    def limits(self) -> tuple[float, float]:
         """Override the limits attribute"""
         return self._get_epics_limits()
 
-    def _get_epics_limits(self):
+    def _get_epics_limits(self) -> tuple[float, float]:
         limits = self.user_setpoint.limits
         if limits is None or limits == (None, None):
             # Not initialized
             return (0, 0)
         return limits
 
+    @limits.setter
```

Comment:
Setters that put to PVs have been divisive in the past...
I think in this case the utility of it is justified and am on board

Reply:
I agree, it's a lot less annoying than requiring some other method that every motor-like needs for changing the limits value.